src/data/data_surveys.tsx

```diff
@@ -0,0 +1,219 @@
+        fav_project: 'Optical Heartbeat Sensor in 20008. I used in infrared sensor for heartbeat detection, a couple of filters to even the signal out, inverters, and comparators to light up an LED for each heartbeat. It was suuper cool!',
+        fav_class: 'ECE 440 (Transmission of Information)! In our second lab, we were able to tune into different various radio stations around West Lafayette using USRPs. It actually puts knowledge of 301 (Signals and Systems) and 302 (Probabilistic Methods in Computer Engineering) to actual use, which made me happy that I could see a practical use of the things I learned in previous classes!',
+        why_ece: "For me, I like the flexibility of ECE, while studying topics that aren't like your stereotypical engineering. Although this is one of the hardest majors, some of the people I've met around really make it worth it. There's a lot of opportunity in ECE if you're willing to go for it!"
+    },
+    {
+        name: 'Altay Atik',
+        fun_fact: 'I moderated the Spark Challenge twice!',
+        advice: 'Manage your time wisely, do not just focus on academics, have a balanced life.',
+        fav_project: 'ECE 20007 Final Project.',
+        fav_class: 'ECE 270 because I like digital systems and how to design them.',
+        why_ece: 'Electrical Engineering was always a passion and a dream for me since I was a kid. I attended multiple robotics competitions and project fairs.'
+    },
+    {
+        name: 'Alyssa Delouise',
+        fun_fact: 'I love to play guitar!',
+        advice: 'Meet lots of your peers and find your study buddies. They make the hard days so much more enjoyable.',
+        fav_project: "Okay I'm like a baby in the major so in ECE???? Uh that final proj for 20007 kinda went hard!!!!",
+        fav_class: '20007/8 because I am new to breadboarding and am having so much fun learning.',
+        why_ece: 'I always knew I wanted to be an engineer, and I found such excitement in the physical aspects of EE and everything I could possibly do at Purdue with EE.'
+    },
+    {
+        name: 'Denae Galloway',
+        fun_fact: 'I played clarinet for 7 years.',
+        advice: 'Never give up, your hard work pays off in the end.',
+        fav_project: '',
+        fav_class: '',
+        why_ece: ''
+    },
+    {
+        name: 'Justin Zhang',
+        fun_fact: 'I can do an ollie 15% of the time.',
+        advice: 'Just like practicing your ollies, you can only get good at ECE if you practice. You may not like it, but that is what peak success in ECE looks like.',
+        fav_project: '',
+        fav_class: '',
+        why_ece: ''
+    },
+    {
+        name: 'Matthew Wen',
+        fun_fact: 'I spend around like 200 hours a year on Spotify. Such a great rate of return.',
+        advice: 'There are nice people in ECE. Avoid the people with superiority complexes.',
+        fav_project: '',
```

Comment:
If it is an undefined property, you can just delete it. Javascript can handle the rest.